engine/src/main/java/org/terasology/rendering/dag/nodes/ChunksRefractiveReflectiveNode.java

```diff
@@ -97,4 +102,11 @@ private void postRenderCleanupSceneReflectiveRefractive() {
         }
         bindDisplay();
     }
+
+    @Override
+    public void update() {
+        // TODO: renames, maybe?
+        FBO sceneOpaque = dynamicFBM.getFBO(DefaultDynamicFBOs.ReadOnlyGBuffer.getResourceUrn());
+        sceneOpaque.attachDepthBufferTo(dynamicFBM.getFBO(REFRACTIVE_REFLECTIVE_URN));
```

Comment:
TODO would be better for now to not lose the general picture and get this PR merged. I'll post a short & sweet PR for this one regarding to first option.